src/main/java/co/novu/api/workflowoverrides/WorkflowOverrideApi.java

```diff
@@ -0,0 +1,48 @@
+package co.novu.api.workflowoverrides;
+
+import co.novu.api.workflowoverrides.pojos.WorkflowOverride;
+
+import co.novu.api.workflowoverrides.request.CreateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.request.UpdateWorkflowOverrideRequest;
+import co.novu.api.workflowoverrides.response.BulkWorkflowOverridesResponse;
+import co.novu.api.workflowoverrides.response.WorkflowOverrideResponse;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.PUT;
+import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
+
+import java.util.Map;
+
+public interface WorkflowOverrideApi {
+
+    String ENDPOINT = "workflow-overrides";
+
+    @POST(ENDPOINT)
+    Call<WorkflowOverride> createWorkflowOverride(@Body CreateWorkflowOverrideRequest request);
+
+    @GET(ENDPOINT)
+    Call<BulkWorkflowOverridesResponse> getWorkflowOverrides(@QueryMap Map<String, Object> params);
+
+    @GET(ENDPOINT + "/workflows/{workflowId}/tenants/{tenantId}")
+    Call<WorkflowOverrideResponse> getWorkflowOverride(@Path("workflowId") String workflowId, @Path("tenantId") String tenantId);
+
+
+    @GET(ENDPOINT + "/{overrideId}")
+    Call<WorkflowOverrideResponse> getWorkflowOverrideById(@Path("overrideId") String overrideId);
+
+
+    @PUT(ENDPOINT + "/{overrideId}")
+    Call<WorkflowOverrideResponse> updateWorkflowOverrideById(@Path("overrideId") String OverrideId, @Body UpdateWorkflowOverrideRequest request);
+
+
+    @PUT(ENDPOINT + "/workflows/{workflowId}/tenants/{tenantId}")
+    Call<WorkflowOverrideResponse> updateWorkflowOverride(@Path("workflowId") String workflowId, @Path("tenantId") String tenantId,
+        @Body UpdateWorkflowOverrideRequest request);
+
+    @DELETE(ENDPOINT + "/{overrideId}")
+    Call<WorkflowOverrideResponse> deleteWorkflowOverride(@Path("overrideId") String overrideId);
```

Comment:
This response body still doesn't conform with what is in the doc.

Reply:
@mayorJAY Actually in the doc nothing is mentioned for the delete API. You should check that. No example is given. So I just added it as an assumption. Can you please check that?

Reply:
<img width="855" alt="Screenshot 2023-12-30 at 5 03 52 AM" src="https://github.com/novuhq/novu-java/assets/70262922/2916d3b3-c3f6-419c-a923-02e79ae32fa5">

Please see this image for the referece.

Reply:
Do I just need to add this i the response body :

{
  boolean: true
}
because I saw this response i other documentation

Reply:
Yes

Reply:
You can use the Swagger doc as reference https://api.novu.co/api#/Workflows-Overrides